app/views/people/index.html.haml

```diff
@@ -38,3 +38,5 @@
         %h4
           = t('.send_invite')
         = render "shared/invitations"
+
+= render "report/report_modal"
```

Comment:
Files should end with a trailing newline

Reply:
If you use an editor/IDE with editorconfig-support or with the respective plugin (https://editorconfig.org/), the editor would add the required newline automatically for you (https://github.com/diaspora/diaspora/blob/develop/.editorconfig#L5) :smiley: